doc/Journal.tex

```diff
@@ -220,143 +220,113 @@ \section{Background}
+Fateley et al.\cite{Fateley00} introduced the use of frequency information to identify spatial position information.
+% Each of which methods? The preceeding and following sentences seem a bit disjointed, but I think the goal is to say that Fateley introduced the idea of using frequency information for REM design which was based on Hadamard matrices. I didn't want to reword in case that reading was incorrect.  
 Each of these methods is based on using a Hadamard matrix for the mask design.
 
-\section{Basic Setup}
+\section{RSM Design}
+\subsection{Basic Model Setup}
+Logan et al.\cite{Logan2017} showed statistical agreement between experimental and simulated DRCs using GEANT4.
+% need to insert GEANT 4 citation  
+Thus, this work will use MCNP to simulate DRCs to evaluate the RSM performance for the designed developed.  
+% need to add MCNP reference; maybe also worth noting  why the switch to MCNP was made
+Instead of using just the full energy peak (FEP), the DRC is formed by adding all counts above 200keV to increase the efficiency of the determination of the source direction.  
+This value was chosen as Logan et al. noted discrepancies for counts below approximately 200keV due to scatter in the  environmental elements not considered in the model \cite{Logan2017}.
+% comment: I didn't realize that you looked at the compton continuum with this work. I have went and modified some of the preceeding sections where I indicated FEP was used. 
+
 Originally, both the analysis of FitzGerald's RSM and the new designs were to be discretized into 10$^\circ$ increments in $\theta$ and 5$^\circ$ in $\phi$.  
 However, due to requirements for the Hadamard method, (which is discussed later) 
 the proposed designs are broken into 32 discrete angles in $\theta$ resulting in $\Delta\theta=11.25^\circ$ and $\Delta\phi=5.625^\circ$ for 30 angles in $\phi$.
 
-The RSM design is to be optimized for a Cs137 point source located at 36 inches from the center of the detector, mimicking Logan et. al's setup.  
+The RSM design is to be optimized for a $^{137}$Cs point source located at 36 inches from the center of the detector, mimicking Logan et. al's setup \cite{Logan2017}.  
 To simulate the relative source rotation in MCNP, the mask is stationary, while the source is rotated in spherical coordinates every degree for $\theta$ from 0 to $348.75^\circ$ and for each $\phi$ from 5.625$^\circ$ to $168.75^\circ$.  
 The modeled NaI detector includes a 1/8 inch 2024 Aluminum alloy sleeve on which the acrylic RSM is placed. 
-The maximum diameter of the RSM depends on the methodology, but the maximum mask thickness is a constant 7.87 inches (20cm).  
+The maximum diameter of the RSM depends on the methodology, but the maximum mask thickness is a constant 7.87 inches (20 cm).  
```

Comment:
Use the word width instead of diameter?  It's not a sphere.

Reply:
Done